rust/datafusion/tests/sql.rs

```diff
@@ -345,6 +345,50 @@ async fn csv_query_group_by_int_min_max() -> Result<()> {
     Ok(())
 }
 
+#[tokio::test]
+async fn csv_query_group_by_float32() -> Result<()> {
+    let mut ctx = ExecutionContext::new();
+    register_aggregate_floats_csv(&mut ctx)?;
+
+    let sql =
+        "SELECT COUNT(*) as cnt, c1 FROM aggregate_floats GROUP BY c1 ORDER BY cnt DESC";
+    let actual = execute(&mut ctx, sql).await;
+
+    // @todo Count OR GROUP BY has a bug?
```

Comment:
@andygrove @alamb @mqy 

Any ideas why COUNT not working with only one value?

Thanks

Reply:
Fixed, forget about CSV header...